packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -116,7 +120,8 @@ class RunCommand extends RunCommandBase {
       RunAndStayResident runner = new RunAndStayResident(
         deviceForCommand,
         target: target,
-        debuggingOptions: options
+        debuggingOptions: options,
+        useDevFS: argResults['devfs']
```

Comment:
This option will go away once we've switched over to the devfs more permanently.